### DISCUSSION TITLE
What are the differences between deepAR and deepastate models?

### DISCUSSION THREAD

**Discussion by mohammednaseefop** [awslabs/gluonts#2169] [Q&A]
What are the differences between deepar and deepstate models?.
What is the criterion for choosing one among them?. What advantages or disadvantages do they have against each other regarding data requirements?.
Are there any cases one model is preferred over the other?

**Comment by rshyamsundar** [comment]
The differences between DeepAR and DeepState are largely similar to those of between the classical Auto-regressive (AR) models and ETS models. So the criteria to choose between DeepAR and DeepState is similar to that of between AR and ETS models:
* no/less prior structure (AR) vs availability of structure (seasonality, trend etc.) that one wants to incorporate explicitly (ETS)
* noise-level in the data: ETS models are relatively more robust to noise in the data compared to AR models
* prediction for very long horizons: The error accumulated by AR models in multi-step forecast setting significantly affects its accuracy.

You choose DeepAR/DeepState over AR/ETS if there are some global effects inherent in your dataset that if learned would improve forecasts for individual time series.
